test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperFormExtensionsTest.cs

```diff
@@ -376,6 +457,49 @@ public void BeginFormWithMethodParameter_CallsHtmlGeneratorWithExpectedValues(Fo
         }
 
         [Theory]
+        [MemberData(nameof(MethodSuppressAntiforgeryAndHtmlAttributesDataSet))]
+        public void BeginFormWithMethodSuppressAntiforgeryAndHtmlAttributesParameters_CallsHtmlGeneratorWithExpectedValues(
```

Comment:
Rather than updating the data sets, suggest reusing the existing data in two separate methods and avoid the conditional `Setup()` below. Pass `suppressAntiforgery: false` in one and `suppressAntiforgery: true` in the other. Same for the other two new tests.